lexicons/com/atproto/admin/searchAccounts.json

```diff
@@ -0,0 +1,45 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.searchAccounts",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get list of accounts that matches your search query.",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "email": { "type": "string" },
+          "limit": { "type": "integer" },
+          "cursor": { "type": "string" }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["accounts"],
+          "properties": {
+            "cursor": { "type": "string" },
+            "accounts": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#accountSearchResult"
+              }
+            }
+          }
+        }
+      }
+    },
+    "accountSearchResult": {
+      "type": "object",
+      "required": ["did"],
+      "properties": {
+        "did": { "type": "string" },
+        "email": { "type": "string" },
+        "normalizedEmail": { "type": "string" },
+        "handle": { "type": "string" }
+      }
+    }
```

Comment:
What would you think of serving a `com.atproto.admin.defs#accountView`, keeping it consistent with similar endpoints like `admin.getAccountInfos`?

Reply:
yep, makes sense to serve accountView if we get rid of normalizedEmail